ash/src/vk/const_debugs.rs

```diff
@@ -2,24 +2,32 @@ use crate::vk::bitflags::*;
 use crate::vk::definitions::*;
 use crate::vk::enums::*;
 use std::fmt;
-pub(crate) fn debug_flags(
+pub(crate) fn debug_flags<
+    T: Copy
+        + fmt::Binary
+        + num::Num
+        + PartialEq
+        + std::ops::BitAnd<Output = T>
+        + std::ops::BitAndAssign
+        + std::ops::Not<Output = T>,
+>(
```

Comment:
Arg I don't like this very much. Especially bringing a new dependency in just for this case. I guess you could do `Flag: Into<u64>`, although you would have to cast inside the loop here I think.

Reply:
Yes, I agree. `Into<u64>` works just as well and requires less code nor the num dependency :grinning: